README.md

```diff
@@ -432,6 +454,9 @@ This project is licensed under the terms of the Apache 2.0 license. See the [LIC
 [badge-mint-on-black]: images/badge_mint_on_black.png
 [badge-black-on-white]: images/badge_black_on_white.png
 [badge-white-on-black]: images/badge_white_on_black.png
+[lockup-black]: images/lockup_black.png
+[lockup-white]: images/lockup_white.png
+[lockup-mint]: images/lockup_mint.png
```

Comment:
Please update the example PNG for the mint lockup to have a dark background. Both the white and the mint lockups are intended to be used only on a dark background. Refer to https://www.afterpay.com/en-US/retailer-resources

Reply:
Thanks for bringing this to my attention - addressed in 87545e1